localstack/services/events/provider_v2.py

```diff
@@ -35,15 +60,80 @@ def create_event_bus(
         tags: TagList = None,
         **kwargs,
     ) -> CreateEventBusResponse:
-        event_bus_arn = f"arn:aws:events:{context.region}:{context.account_id}:event-bus/{name}"
-        event_bus = {"Name": name, "Arn": event_bus_arn}
-        self._event_buses[name] = event_bus
-
+        region = context.region
+        account_id = context.account_id
+        store = self.get_store(context)
```

Comment:
The `get_store` util is quite convenient and helps to keep the behavior consistent 👍